user-guide/docs/invoking-your-first-function-with-js-sdk.md

```diff
@@ -1,192 +1,176 @@
-> Once completed the User can invoke a conclave cloud function from the project ui.
+After completing this tutorial, you will be able to invoke a Conclave Cloud function from the project UI.
 
 ## Prerequisites
 
-In order to follow this tutorial you should already have done the following:
+To follow this tutorial, you should already have done the following:
 
-- Sign-up for a [Conclave Cloud Beta](https://www.conclave.cloud/) account and log in to the portal.
-- Download and install the `ccl` tool for your operating system and verify it
-  works.
-- Upload the simple.js function to [Conclave Cloud Beta](https://www.conclave.cloud/) by following the steps documented in [Creating your first function](creating-your-first-function.md).
+- Sign-up for a [Conclave Cloud](https://www.conclave.cloud/) account and log in to the portal.
+- Download and install the `ccl` tool as per the instructions [here](index.md).
+- Upload the simple.js function to [Conclave Cloud](https://www.conclave.cloud/) as per the instructions [here](creating-your-first-function.md).
 
 ## 1. Run the Hello Conclave Cloud project
 
-Get the [Hello Conclave Cloud Tutorial](https://github.com/R3Conclave/conclave-cloud-samples/) repository:
+* Get the [Hello Conclave Cloud Tutorial](https://github.com/R3Conclave/conclave-cloud-samples/) repository:
 
-```
-git clone https://github.com/R3Conclave/conclave-cloud-samples.git
-```
+  ```
+  git clone https://github.com/R3Conclave/conclave-cloud-samples.git
+  ```
 
-In the `hello-conclave-cloud-tutorial` directory install and run the project:  
-`npm install`  
-`npm start`
+* Change to the `hello-conclave-cloud-tutorial` directory.
 
-_This is the base ui set-up for the sayHello function call, the sdk has yet to be configured._
+* Install and run the project:  
+  `npm install`  
+  `npm start`
 
-## 2. Install and Configure the Conclave Cloud SDK
+  _This is the base UI set-up for the sayHello function call. The SDK is yet to be configured._
 
-Install the conclave-cloud-sdk npm package:  
-`npm install /path to file/conclave-cloud-sdk-1.0.0-beta2.tgz --save`
+## 2. Install and configure the Conclave Cloud SDK
 
-In the projects root directory create a new file and call it `ConclaveCloud.js`  
-Add the following configurations:
+* Install the conclave-cloud-sdk npm package:  
+  `npm install /path to file/conclave-cloud-sdk-1.0.0-beta2.tgz --save`
 
-1. _ConclaveCloud.ts_
-```
-import { Conclave } from 'conclave-cloud-sdk'
+* Create a project.
 
-const accessToken: ()=>Promise<String>=...;
+* In the project's root directory, create a new file and call it `ConclaveCloud.js`.
 
-const conclaveConfig = new Conclave.create({
-    projectUid: 'projectUid',
-    accessToken
-});
+* Create an API key using the Conclave Cloud CLI command line tool:
+  `ccl apikeys create`
 
-export default conclaveConfig;
-```
+* Add the following configurations to `ConclaveCloud.js`
+   ```
+   export interface ConclaveClientConfig {
```

Comment:
This here will not work as is. It should be:

````
import { Conclave } from 'conclave-cloud-sdk'

const conclaveConfig = new Conclave.create({
    apiKey: 'apiKey' // place the apiKey you've created here
});
export default conclaveConfig;
````

Reply:
Thanks Mihail!